src/ansys/geometry/core/_grpc/_services/v0/repair_tools.py

```diff
@@ -434,3 +455,91 @@ def serialize_issue(issue):
                 for body_issues in response.issues_by_body
             ]
         }
+
+    @staticmethod
+    def serialize_tracker_command_response(response) -> dict:
+        """Serialize a TrackerCommandResponse object into a dictionary.
+
+        Parameters
+        ----------
+        response : TrackerCommandResponse
+            The gRPC TrackerCommandResponse object to serialize.
+
+        Returns
+        -------
+        dict
+            A dictionary representation of the TrackerCommandResponse object.
+        """
+
+        def serialize_body(body):
+            return {
+                "id": body.id,
+                "name": body.name,
+                "can_suppress": body.can_suppress,
+                "transform_to_master": {
+                    "m00": body.transform_to_master.m00,
+                    "m11": body.transform_to_master.m11,
+                    "m22": body.transform_to_master.m22,
+                    "m33": body.transform_to_master.m33,
+                },
+                "master_id": body.master_id,
+                "parent_id": body.parent_id,
+            }
+
+        def serialize_entity_identifier(entity):
+            """Serialize an EntityIdentifier object into a dictionary."""
+            return {
+                "id": entity.id,
+                "type": entity.type,
+            }
+
+    @staticmethod
+    def serialize_tracker_command_response(response) -> dict:
```

Comment:
Method shadowing is happening -- only the second implementation will be used. Why do we have it duplicated? Can we get rid of the one that is not used?

Also.. can we avoid making it a "staticmethod" and make it a normal, protected method? (i.e. ``_serialize_tracker_command_response``)